src/libs/actions/Report.js

```diff
@@ -551,15 +551,19 @@ function updateReportActionMessage(reportID, sequenceNumber, message) {
             return;
         }
 
-        // If the message is deleted, update the last read in case the deleted message is being counted in the unreadActionCount
+        // If the message is deleted, we should
+        // 1. update the last read in case the deleted message is being counted in the unreadActionCount
```

Comment:
Capitalize `Update`?

Also, I don't entirely understand this comment "in case the deleted message is being counted". Can we tell if the deleted message is counted in the `unreadActionCount` or not? This comment reads like "maybe we need to do this".

Reply:
Seems like we are accounting for deleted messages - https://github.com/Expensify/App/blob/main/src/libs/actions/Report.js#L116. I'll update the comment.

Reply:
It seems that we are accounting for deleted messages [here](https://github.com/Expensify/App/blob/main/src/libs/actions/Report.js#L116). I'll update that comment.

Reply:
It seems that we are not counting deleted messages in `unreadActionCount`, but we want to update to update the last read and the counter anyways. I updated the comment to make it clear.